src/RetryPolicy.test.ts

```diff
@@ -178,4 +178,16 @@ describe('RetryPolicy', () => {
     await expect(policy.execute(s)).to.eventually.be.rejectedWith(MyErrorA);
     expect(onGiveUp).to.have.been.calledWith({ error: err });
   });
+
+  it('provides the attempt to the onRetry callback', async () => {
+    const s = stub().throws(new MyErrorA());
+    const attempts: number[] = [];
+    const policy = retry(handleAll, { maxAttempts: 3 });
+    policy.onRetry(({ attempt }) => {
+      attempts.push(attempt);
+    });
+
+    await expect(policy.execute(s)).to.eventually.be.rejectedWith(MyErrorA);
+    expect(attempts).to.deep.equal([1, 2, 3]);
```

Comment:
The documentation says attempts should start at `0`, but it starts at `1` here. I don't feel strongly either way but one of the two should be updated.

Reply:
Or maybe the documentation is just a little bit unclear. I would say "starting at 1" since 0 doesn't actually get emitted in the retry.

Reply:
I'll adapt the docs

Reply:
done